docs/git-workflow.md

```diff
@@ -1,36 +1,67 @@
 
-This document describes the git workflow that *all* developers of the narrative code must follow. It also describes the release procedure, which only an unlucky few should have to worry about.
+- Last modified: Aug. 31, 2020
+
+***Table of Contents***
+
+- [Branches and Tags](#branches-tags)
+- [Contributing](#Contributing)
+  - [Pull Requests](#pull-requests)
+- [Release Flows](#realease-flow)
+  - [Development Releases](#development-releases)
+  - [Production Releases](#production-releases)
+  - [Hotfix Releases](#hotfixes-releases)
 
 ## Branches and Tags
 
 There are 3 branches:
 
-* `develop` – Development branch. New features are added off this branch.
-* `staging` – Staging or "release candidate" branch. The branch deployed `narrative-dev`.
-* `master` – Production branch. The branch deployed on `narrative-next` and `narrative`. This branch is used for hotfixes.
+- `develop` – Development branch. New features are added off this branch.
+- `staging` – Staging or "release candidate" branch. The branch deployed `narrative-dev`.
+- `master` – Production branch. The branch deployed on `narrative-next` and `narrative`. This branch is used for hotfixes.
 
 Tags on master are used for indicating releases (see "Production releases", below).
 
-## Workflow
+## Contributing
 
-There are three main workflows: development, production releases, and hotfixes.
+To contribute to this repository create a pull request for a new feature or issue.
+
+### Pull Requests
+
+#### _Internal Developers_
+
+To begin working on a feature or story, make sure you've assigned yourself to the story/issue in JIRA and marked it as "In Progress". If there isn't an story/issue yet for what you want to work on, please create one.
+
+Create a new branch off `develop` using the naming convention:
+
+`{your initials or username}-{summary}-{JIRA #}`
+
+For example: `jd-data-panel-button-112`
 
-### Development
+When your branch is ready for review, open a PR into `develop` and request reviews from relevant team members. Address any failing tests, lint errors, PR feedback, etc. Once the work is approved, it will be merged.
 
-To add a feature or fix a bug in development, work from the develop branch, i.e. create a new branch off develop and do a pull request when done (or directly push it back).
+More detailed directions for the feature branch workflow are located on [Github](https://guides.github.com/introduction/flow/)
+
+#### _External 3rd Party Developers_
+
+External developers do not have write access to the repository. To create a pull request, fork the repository and submit a pull request from the fork. Instructions for how to fork a repository are located on [GitHub](https://guides.github.com/activities/forking/).
+
+## Release Flows
+
+There are three main workflows: development, production releases, and hotfixes.
```

Comment:
Once we hear from Jason tomorrow, it may be that this whole section can get the chop, or at least be heavily revised.

Reply:
I agree, but it should be in a separate branch/JIRA ticket so we don't get any more PR bloat. :)